camayoc/tests/qcs/cli/conftest.py

```diff
@@ -24,12 +24,19 @@ def qpc_server_config():
     port = config.get('qcs', {}).get('port')
     username = config.get('qcs', {}).get('username', 'admin')
     password = config.get('qcs', {}).get('password', 'pass')
+    ssl_verify = config.get('qcs', {}).get('ssl-verify', False)
```

Comment:
You may ask about the `https` value in the config file. The difference is that https controls if the scheme for the URL should be either http or https. ssl-verify on the other hand defines if the connection should be verified or not True/False or if a custom certificate should be used (when the ssl-verify is a path to the certificate file).

The API client can also leverage these two config options to play with adding http or https to the URL and configuring requests' verify argument.

Reply:
OK, that makes sense!